global.json

```diff
@@ -1,6 +1,7 @@
 {
     "sdk": {
-        "rollForward": "latestMajor",
+        "version": "3.1.100",
```

Comment:
Why not 3.1.404?

Reply:
Because it would require everyone to have that specific version. With 3.1.100 folks can have that or any one from and up to 3.1.404.